submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/stats/web/StatsResource.kt

```diff
@@ -0,0 +1,56 @@
+package ac.uk.ebi.biostd.stats.web
+
+import ac.uk.ebi.biostd.stats.web.handlers.StatsFileHandler
+import ac.uk.ebi.biostd.submission.domain.service.TempFileGenerator
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PatchMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.multipart.MultipartFile
+import uk.ac.ebi.stats.model.SubmissionStat
+import uk.ac.ebi.stats.model.SubmissionStatType
+import uk.ac.ebi.stats.service.SubmissionStatsService
+
+@RestController
+@RequestMapping("/stats")
+class StatsResource(
```

Comment:
in case you are add IT for this, please ping me I think we need mvc test mock based instead of full integration test

Reply:
Done